libsql-server/src/http/user/listen.rs

```diff
@@ -103,42 +138,57 @@ async fn listen_stream(
     actions: Option<Vec<Action>>,
 ) -> impl Stream<Item = crate::Result<AggregatorEvent>> {
     async_stream::try_stream! {
-        let _sub = Subscription {
-            store: store.clone(),
-            namespace: namespace.clone(),
-            table: table.clone(),
-        };
-
-        let mut stream = store.subscribe(namespace.clone(), table.clone());
+        let _sub = Subscription::new(store.clone(), namespace.clone(), table.clone());
+        let mut stream = store.subscribe(namespace, table);
 
         while let Some(item) = stream.next().await  {
             match item {
-                Ok(msg) => if filter_actions(&msg, &actions) {
+                Ok(msg) if filter_actions(&msg, &actions) => {
                     LISTEN_EVENTS_SENT.increment(1);
                     yield AggregatorEvent::Changes(msg);
                 },
                 Err(BroadcastStreamRecvError::Lagged(n)) => {
                     LISTEN_EVENTS_DROPPED.increment(n as u64);
-                    yield AggregatorEvent::Error(&LAGGED_MSG);
+                    yield AggregatorEvent::Error(LAGGED_MSG);
                 },
+                _ => {}
```

Comment:
```suggestion
                Err(e) => {
                    yield Err(Error::Internal(format!("Unexpected error in listen stream: {}", e)))?;
                },
```

@athoscouto would this work?

Reply:
let's log them as well

Reply:
Maybe just warn them

Reply:
I pushed an update, let me know if that works.